backpack/__init__.py

```diff
@@ -215,7 +219,7 @@ def hook_run_extensions(
         memory_cleanup(module)
 
 
-def extend(module: Module, debug: bool = False) -> Module:
+def extend(module: torch.nn.Module, debug=False, use_converter=False):
```

Comment:
Add types (is this file part of `fully_documented`? I would expect the docstring linter to complain about missing types here)

Reply:
Yes, it is. But I did not add a type hint checker yet.